### PR TITLE
enhancement(Webhooks): quick design update

### PR DESCRIPTION
**Pull request by Betree**
- styled-components => Tailwind
- Use the latest OC standard design practices:
  - Toast for error
  - Buttons positions
  - Empty view with icon

| Before | After |
|--------|--------|
| image | image |
| image | image |

### CODE REVIEW

File: components/edit-collective/sections/Webhooks.js [hunk 1]
```diff
@@ -170,62 +167,58 @@ class Webhooks extends React.Component {
...
-          </Box>
-          <Box>
-            <Label fontSize="14px" mb={1}>
+          </div>
+          <div>
+            <label htmlFor={`event-type-select-${index}`} className="text-sm font-medium">
               <FormattedMessage defaultMessage="Activity" id="ZmlNQ3" />
-            </Label>
-            <Flex alignItems="center">
+            </label>
```
Comment (gustavlrsn) [inline components/edit-collective/sections/Webhooks.js:216]: There is a shadcn/ui Label in `ui/Label.tsx` we could use here

File: components/edit-collective/sections/Webhooks.js [hunk 2]
```diff
@@ -234,18 +227,18 @@ class Webhooks extends React.Component {
...
+                  variant="ghost"
+                  size="icon"
+                  className="ml-2"
                   title={intl.formatMessage({ id: 'moreInfo', defaultMessage: 'More info' })}
                   onClick={() => this.setState({ moreInfoModal: webhook.type })}
-                  ml={2}
                 >
-                  <InfoCircle size={24} color="#a3a3a3" />
-                </StyledButton>
+                  <InfoCircle size={18} color="#a3a3a3" />
```
Comment (gustavlrsn) [inline components/edit-collective/sections/Webhooks.js:237]:
```suggestion
                  <InfoCircle size={18} className="text-muted-foreground" />
```

File: components/edit-collective/sections/Webhooks.js [hunk 3]
```diff
@@ -295,60 +287,49 @@ class Webhooks extends React.Component {
...
-            borderRadius={6}
-            buttonSize="medium"
-            buttonStyle="standard"
-            css={'border-style: dashed'}
-            onClick={() => this.addWebhook()}
-          >
-            <Add size="1.2em" />
-            {'  '}
+        <div className="mb-6 mt-8 flex items-center justify-between">
+          <h3 className="text-xl font-bold text-gray-900">
```
Comment (gustavlrsn) [inline components/edit-collective/sections/Webhooks.js:295]: The shadcn/ui components is using the `slate` version of grays, and the default text color applied to all elements is `text-slate-950`.

So we could either do
```suggestion
          <h3 className="text-xl font-bold">
```
as 950 is close to 900, or 

```suggestion
          <h3 className="text-xl font-bold text-slate-900">
```
to be specific.

(We might want to remove the regular `gray`s from the available palette, or map them to the `slate` palette..)

File: components/edit-collective/sections/Webhooks.js [hunk 4]
```diff
@@ -295,60 +287,49 @@ class Webhooks extends React.Component {
...
+          </h3>
+          <Button onClick={this.addWebhook}>
+            <PlusCircle className="mr-2 h-5 w-5" /> <FormattedMessage defaultMessage="New Webhook" id="q7eF+t" />
+          </Button>
+        </div>
+
+        {webhooks.length === 0 ? (
+          <div className="rounded-lg border bg-card py-12 text-center text-card-foreground shadow-sm">
+            <WebhookIcon className="mx-auto mb-4 h-16 w-16 text-gray-400" />
+            <h4 className="text-lg font-semibold text-gray-900">
```
Comment (gustavlrsn) [inline components/edit-collective/sections/Webhooks.js:310]: Same thing here
```suggestion
            <h4 className="text-lg font-semibold">
```

**Review by gustavlrsn — APPROVED** [review]
Excellent update, looks much better!

**Comment by Betree** [comment]
@gustavlrsn thanks for the review, I've applied your suggestions!

**Pull request merged**